doc/POST_INSTALL.md

```diff
@@ -1,7 +1,7 @@
-This is a dummy disclaimer to display after the install
+The app url is `__DOMAIN____PATH__`. You will not need this in the future, the runner will contact Peertube directly.
 
-The app url is `__DOMAIN____PATH__`
+The app install dir is `__INSTALL_DIR__`.
 
-The app install dir is `__INSTALL_DIR__`
+The app id is `__ID__`.
```

Comment:
This is confusing and looks like the dummy / example notification.md ... If this ain't useful, then no need to keep it around x_x